docs/user/reference/features.rst

```diff
@@ -0,0 +1,9 @@
+========
+Features
```

Comment:
🤩

Reply:
Taking shape! It should maybe become `user/reference/features/index.rst` but that can always happen.

Reply:
We have an outstanding item to think a bit more about how features should be structured, so I wanted the simplest approach.